fuzz/fuzz_targets/deserialize.rs

```diff
@@ -0,0 +1,46 @@
+#![no_main]
+#[macro_use]
+extern crate libfuzzer_sys;
+extern crate parity_wasm;
+extern crate mktemp;
+
+use std::fs::File;
+use std::io::Write;
+use std::path::PathBuf;
+use std::process::Command;
+
+fn wasm_opt() -> PathBuf {
+    let bin = PathBuf::from(env!("OUT_DIR")).join("bin").join("wasm-opt");
+    assert!(
+        bin.exists(),
+        format!("could not find wasm-opt at: {:?}", wasm_opt())
+    );
+    bin
+}
+
+fuzz_target!(|data: &[u8]| {
+    let seed = mktemp::Temp::new_file().unwrap();
```

Comment:
could you please use `.expect(<Possible reason for failure>)` instead of unwrap here and in few other places

(this is more friendly to error diagnostic, just in case)

Reply:
fixed in b318925